### ISSUE TITLE
Nothing appears when editing article

### ISSUE THREAD

**Issue by caleb15** [forem/forem#6927]
**Describe the bug**

Sometimes when editing a article nothing appears.

**To Reproduce**

1. Edit a article
2. Nothing appears

**Expected behavior**
Something to appear

**Screenshots**

https://i.imgur.com/OFSt2Wt.png

**Desktop (please complete the following information):**

OS: Win10
Browser: Chrome
Version: Version 80.0.3987.149 (Official Build) (64-bit) ~ latest version

**Additional context**

It happens even after refreshing the page. If I go back to the dashboard page however and refresh there the problem goes away for the article I clicked on. Also as soon as it happens to one article all the other articles start having the same problem.

Console log:
```
VM793 articleForm-7d87d9911f46e5ed5529.chunk.js:1 Uncaught (in promise) TypeError: Cannot read property 'innerHTML' of null
    at new o (VM793 articleForm-7d87d9911f46e5ed5529.chunk.js:1)
    at L (VM783 vendor-d130b36e9d4353fba56b.chunk.js:2)
    at VM783 vendor-d130b36e9d4353fba56b.chunk.js:2
    at A (VM783 vendor-d130b36e9d4353fba56b.chunk.js:2)
    at C (VM783 vendor-d130b36e9d4353fba56b.chunk.js:2)
    at B (VM783 vendor-d130b36e9d4353fba56b.chunk.js:2)
    at VM793 articleForm-7d87d9911f46e5ed5529.chunk.js:1
o @ VM793 articleForm-7d87d9911f46e5ed5529.chunk.js:1
L @ VM783 vendor-d130b36e9d4353fba56b.chunk.js:2
(anonymous) @ VM783 vendor-d130b36e9d4353fba56b.chunk.js:2
A @ VM783 vendor-d130b36e9d4353fba56b.chunk.js:2
C @ VM783 vendor-d130b36e9d4353fba56b.chunk.js:2
B @ VM783 vendor-d130b36e9d4353fba56b.chunk.js:2
(anonymous) @ VM793 articleForm-7d87d9911f46e5ed5529.chunk.js:1
Promise.then (async)
Ee @ VM793 articleForm-7d87d9911f46e5ed5529.chunk.js:1
(anonymous) @ VM793 articleForm-7d87d9911f46e5ed5529.chunk.js:1
s @ base-4809c40c0a403b5523befd7c31c6ee752e01759c0dfc40a1e9a1a83da5c35188.js:6
c @ base-4809c40c0a403b5523befd7c31c6ee752e01759c0dfc40a1e9a1a83da5c35188.js:6
T @ base-4809c40c0a403b5523befd7c31c6ee752e01759c0dfc40a1e9a1a83da5c35188.js:6
p @ base-4809c40c0a403b5523befd7c31c6ee752e01759c0dfc40a1e9a1a83da5c35188.js:6
e.___hb.e.___hb @ base-4809c40c0a403b5523befd7c31c6ee752e01759c0dfc40a1e9a1a83da5c35188.js:4
base-4809c40c0a403b5523befd7c31c6ee752e01759c0dfc40a1e9a1a83da5c35188.js:4 
-oooooooo/-      .+ooooooooo:  +ooo+        oooo/
+MMMMMMMMMMm+   -NMMMMMMMMMMs  +MMMM:      /MMMM/
+MMMNyyydMMMMy  /MMMMyyyyyyy/   mMMMd      mMMMd
+MMMm    :MMMM. /MMMN           /MMMM/    /MMMM:
+MMMm    .MMMM- /MMMN            dMMMm    mMMMh
+MMMm    .MMMM- /MMMMyyyy+       :MMMM/  +MMMM-
+MMMm    .MMMM- /MMMMMMMMy        hMMMm  NMMMy
+MMMm    .MMMM- /MMMMoooo:        -MMMM+oMMMM-
+MMMm    .MMMM- /MMMN              yMMMmNMMMy
+MMMm    +MMMM. /MMMN              .MMMMMMMM.
+MMMMdddNMMMMo  /MMMMddddddd+       sMMMMMMs
+MMMMMMMMMNh:   .mMMMMMMMMMMs        yMMMMs
.///////:-        -/////////-         .::.

Hey there! Interested in the code behind dev.to? Well, you're in luck - we're open source!

Come say hi, tell us what you're debugging, or even lend a hand in our repo: https://github.com/thepracticaldev/dev.to

Did you find a bug or vulnerability? Check out our bug bounty info here: https://dev.to/security

VM793 articleForm-7d87d9911f46e5ed5529.chunk.js:1 Uncaught (in promise) TypeError: Cannot read property 'innerHTML' of null
    at new o (VM793 articleForm-7d87d9911f46e5ed5529.chunk.js:1)
    at L (VM783 vendor-d130b36e9d4353fba56b.chunk.js:2)
    at VM783 vendor-d130b36e9d4353fba56b.chunk.js:2
    at A (VM783 vendor-d130b36e9d4353fba56b.chunk.js:2)
    at C (VM783 vendor-d130b36e9d4353fba56b.chunk.js:2)
    at B (VM783 vendor-d130b36e9d4353fba56b.chunk.js:2)
    at VM793 articleForm-7d87d9911f46e5ed5529.chunk.js:1
o @ VM793 articleForm-7d87d9911f46e5ed5529.chunk.js:1
L @ VM783 vendor-d130b36e9d4353fba56b.chunk.js:2
(anonymous) @ VM783 vendor-d130b36e9d4353fba56b.chunk.js:2
A @ VM783 vendor-d130b36e9d4353fba56b.chunk.js:2
C @ VM783 vendor-d130b36e9d4353fba56b.chunk.js:2
B @ VM783 vendor-d130b36e9d4353fba56b.chunk.js:2
(anonymous) @ VM793 articleForm-7d87d9911f46e5ed5529.chunk.js:1
Promise.then (async)
Ee @ VM793 articleForm-7d87d9911f46e5ed5529.chunk.js:1
(anonymous) @ VM793 articleForm-7d87d9911f46e5ed5529.chunk.js:1
Promise.then (async)
60 @ VM793 articleForm-7d87d9911f46e5ed5529.chunk.js:1
l @ bootstrap:79
t @ bootstrap:45
r @ bootstrap:32
(anonymous) @ bootstrap:145
(anonymous) @ runtime~articleForm-e5fdcb2de9060f42d464.js:1
```

**Comment by caleb15** [comment]
Note that this is the third bug in a row I've ran into in less than a hour... 😬 
Maybe I've angered a witch or something, talk about bad luck :|

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
> Note that this is the third bug in a row I've ran into in less than a hour... 😬
> Maybe I've angered a witch or something, talk about bad luck :|

hi @caleb15 :-( As I asked in the other two issues, do you have any particular extension installed? Because I can't reproduce either of them :(

**Comment by caleb15** [comment]
Okta, quick javascript switcher, and onetab, none of which should be interacting with the website. 

Okta is for signins, sorta like lastpass.
Quick javascript switcher is for turning off javascript but I have javascript enabled for dev.to. 
Onetab allows you to store and group your tabs. 

This isn't a easy issue to reproduce unfortunately. If you have some sort of error tracking like sentry maybe you can find it there.

**Comment by caleb15** [comment]
Unlike the other issues this one appears reproducible - try doing a hard refresh, clearing your cookies, signing in again, and editing your articles. Eventually you might hit a article that has this error.

**Comment by caleb15** [comment]
I just tried disabling extensions and the error still happened.

**Comment by rhymes** [comment]
hi @caleb15, do you remember if anything appeared in the console? trying to understand if there's some weird JS bug going on... I'll try to reproduce it in the meantime

**Comment by lightalloy** [comment]
I got this error once while trying to reproduce #6926, editing [a post](https://dev.to/lightalloy/improve-your-sleep-as-a-developer-21a9/). This bug is hard to catch/reproduce, as it doesn't happen consistently.
The error was on the line `helpHTML: document.getElementById('editor-help-guide').innerHTML` in the `articleForm.jsx` (I haven't recorded on which one of 2 unfortunately)

**Comment by lightalloy** [comment]
While trying to reproduce this and #6926 I ran into another issue #6949 which could be related to these issues (or not).

**Comment by caleb15** [comment]
@rhymes I included the console log in the original post at the top of this thread, a TypeError appeared in the console.

**Comment by rhymes** [comment]
@lightalloy @caleb15 are you able to reproduce this still?

**Comment by lightalloy** [comment]
I was still able to reproduce it after several attempts, an error showed up in the console:
```
TypeError: document.getElementById(...) is null    // articleForm.jsx:102:25
```

**Comment by cmgorton** [comment]
With the removal of our service worker and fixes with caching I would imagine we should not experience this issue anymore. I am going to close this issue but please let me know if you run in to it again and we can reopen this!
